src/realm/table.hpp

```diff
@@ -394,7 +394,21 @@ class Table {
     /// \sa Table::set_null_unique()
     void merge_rows(size_t row_ndx, size_t new_row_ndx);
 
-    // Get cell values. Will assert if the requested type does not match the column type
+    //@{
+
+    /// Get cell values.
+    /// Will assert if the requested type does not match the column type.
+    ///
+    /// When querying a nullable column and the value is null, a default value
```

Comment:
I know what you mean, but it's not confusing to me. We could `s/querying/fetching from/`.